src/runtime/internal/dom.ts

```diff
@@ -307,6 +330,16 @@ export class HtmlTag {
 		this.n = Array.from(this.e.childNodes);
 	}
 
+	l(nodes: any) {
+		this.n = this.n.map(n => {
+			if (n.nodeType === 3) {
+				return claim_text(nodes, (n as Text).data);
+			 } else {
+				return claim_element(nodes, n.nodeName, n, n.namespaceURI !== 'http://www.w3.org/1999/xhtml');
```

Comment:
`'http://www.w3.org/1999/xhtml'` could be replaced with `namespaces.html`: 
https://github.com/sveltejs/svelte/blob/767ce22ed1b8de41573311dd6180c12837443cc9/src/compiler/utils/namespaces.ts#L1